features/prompt-history/groups.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Groups"
+icon: "ball-pile"
+---
+
+[Endpoint Reference](/reference/track-group)
+
+To associate requests with a group from the prompt registry, run the code
```

Comment:
Groups are not in the prompt registry. Let's change this.

"It is helpful to link requests to eachother when building workflows with chains and agents. PromptLayer allows you to create a group and add individual request IDs into the group. You will then be able to visually inspect groups through the dashboard."

Reply:
Great